PlanPluginBridge/src/main/java/com/djrapitops/pluginbridge/plan/nucleus/NucleusData.java

```diff
@@ -0,0 +1,311 @@
+                    User operatorUser = getUser(muteInfo.getMuter().get());
+                    if (operatorUser != null) {
+                        String operator = operatorUser.getName();
+                        link = Html.LINK.parse(PlanAPI.getInstance().getPlayerInspectPageLink(operator), operator);
+                    }
+                }
+
+                if (muteInfo.getCreationInstant().isPresent()) {
+                    start = FormatUtils.formatTimeStampYear(muteInfo.getCreationInstant().get().toEpochMilli());
+                }
+
+                if (muteInfo.getRemainingTime().isPresent()) {
+                    end = FormatUtils.formatTimeStampYear(muteInfo.getRemainingTime().get().plusMillis(System.currentTimeMillis()).toMillis());
+                } else {
+                    end = "Permanent mute";
+                }
+
+                inspectContainer.addValue("&nbsp;" + getWithIcon("Operator", Icon.called("user").of(Color.DEEP_ORANGE)), link);
+                inspectContainer.addValue("&nbsp;" + getWithIcon("Date", Icon.called("calendar").of(Color.DEEP_ORANGE).of(Family.REGULAR)), start);
+                inspectContainer.addValue("&nbsp;" + getWithIcon("Ends", Icon.called("calendar-check").of(Color.DEEP_ORANGE).of(Family.REGULAR)), end);
+                inspectContainer.addValue("&nbsp;" + getWithIcon("Reason", Icon.called("comment").of(Color.DEEP_ORANGE).of(Family.REGULAR)), reason);
+            }
+        }
+
+        if (NucleusAPI.getJailService().isPresent()) {
+            NucleusJailService jailService = NucleusAPI.getJailService().get();
+
+            boolean jailed = jailService.isPlayerJailed(user);
+            inspectContainer.addValue(getWithIcon("Jailed", Icon.called("bars").of(Color.YELLOW).of(Family.SOLID)), jailed ? "Yes" : "No");
+
+            if (jailed && jailService.getPlayerJailData(user).isPresent()) {
+                Inmate inmate = jailService.getPlayerJailData(user).get();
+
+                String reason = inmate.getReason();
+                String start = "Unknown";
+                String end;
+                String link = "Unknown";
+
+                if (inmate.getJailer().isPresent()) {
+                    User operatorUser = getUser(inmate.getJailer().get());
+                    if (operatorUser != null) {
+                        String operator = operatorUser.getName();
+                        link = Html.LINK.parse(PlanAPI.getInstance().getPlayerInspectPageLink(operator), operator);
+                    }
+                }
+
+                if (inmate.getCreationInstant().isPresent()) {
+                    start = FormatUtils.formatTimeStampYear(inmate.getCreationInstant().get().toEpochMilli());
+                }
+
+                if (inmate.getRemainingTime().isPresent()) {
+                    end = FormatUtils.formatTimeStampYear(inmate.getRemainingTime().get().plusMillis(System.currentTimeMillis()).toMillis());
```

Comment:
Can be simplified with Optional#map
`String end = muteInfo.getRemainingTime().map(instant -> FormatUtils.formatTimeStampYear(instant.toEpochMilli())).orElse("Permanent mute")`

Additionally since this seems to be done often it could be split into it's own method (orElse String as parameter)